tests/test_tron_tools.py

```diff
@@ -1004,3 +1019,38 @@ def test_list_tron_clusters(self, mock_glob):
         ]
         result = tron_tools.list_tron_clusters('foo')
         assert sorted(result) == ['dev-cluster2', 'prod']
+
+
+def test_extract_jobs_from_tron_yaml_with_empty_dict():
+    assert tron_tools.extract_jobs_from_tron_yaml({}) == {}
+
+
+def test_extract_jobs_from_tron_yaml_with_None():
+    assert tron_tools.extract_jobs_from_tron_yaml({"jobs": None}) == {}
+
+
+def test_extract_jobs_from_tron_yaml_with_no_jobs():
+    assert tron_tools.extract_jobs_from_tron_yaml({"jobs": {}}) == {}
+
+
+def test_extract_jobs_from_tron_yaml_with_just_jobs():
+    assert tron_tools.extract_jobs_from_tron_yaml({"job0": "foo"}) == {"job0": "foo"}
+
+
+def test_extract_jobs_from_tron_yaml_with_mix():
+    config = {
+        "_template": "foo",
+        "job0": "bar",
+    }
+    assert tron_tools.extract_jobs_from_tron_yaml(config) == {"job0": "bar"}
+
+
+def test_extract_jobs_from_tron_yaml_defaults_to_jobs_if_available():
+    config = {
+        "_template": "foo",
+        "job0": "bar",
+        "jobs": {
+            "job1": "baz",
+        },
+    }
+    assert tron_tools.extract_jobs_from_tron_yaml(config) == {"job1": "baz:"}
```

Comment:
typo at the end (extra `:`)

Reply:
whoa I swear the tests passed on my machine...
must be a vim artifact..... :)